prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum_multifile/result.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "title": "Add `@@schema` attribute",
+    "kind": "quickfix",
+    "diagnostics": [
+      {
+        "range": {
+          "start": {
+            "line": 0,
+            "character": 0
+          },
+          "end": {
+            "line": 2,
+            "character": 1
+          }
+        },
+        "severity": 1,
+        "message": "Error validating model \"User\": This model is missing an `@@schema` attribute."
+      }
+    ],
+    "edit": {
+      "changes": {
+        "file:///path/to/_target.prisma": [
+          {
+            "range": {
+              "start": {
+                "line": 2,
+                "character": 0
+              },
+              "end": {
+                "line": 2,
+                "character": 1
+              }
+            },
+            "newText": "\n    @@schema()\n}"
+          }
+        ]
+      }
+    }
+  },
+  {
+    "title": "Add `@@schema` attribute",
+    "kind": "quickfix",
+    "diagnostics": [
+      {
+        "range": {
+          "start": {
+            "line": 4,
+            "character": 0
+          },
+          "end": {
+            "line": 6,
+            "character": 1
+          }
+        },
+        "severity": 1,
+        "message": "This enum is missing an `@@schema` attribute."
```

Comment:
Huh, just noticed the side-by-side that the enum validation doesn't include something like:
```
Error validating enum \"Colour\":...
```

Reply:
Is it something that we've lost in this PR?

Reply:
Nah, just something I noticed with the test you added :)